dashboard/app/controllers/api/v1/pd/workshop_survey_report_controller.rb

```diff
@@ -7,6 +7,7 @@ class WorkshopSurveyReportController < ReportControllerBase
     include ::Pd::WorkshopSurveyReportCsvConverter
     include Pd::WorkshopSurveyResultsHelper
     include Pd::SurveyPipeline::Helper
+    include Pd::WorkshopSurveyConstants
```

Comment:
Why the added `include` here, when it looks like we're only removing code?

Reply:
`Pd::WorkshopSurveyConstants` used to be included in `Pd::WorkshopSurveyResultsHelper`.  
`Pd::WorkshopSurveyResultsHelper` no longer needs `Pd::WorkshopSurveyConstants` but the controller still does, so I move it to the controller.